modules/installation-osp-verifying-external-network.adoc

```diff
@@ -0,0 +1,30 @@
+//Module included in the following assemblies:
+//
+// * installing/installing_openstack/installing-openstack-installer.adoc
+
+[id="installation-osp-verifying-external-network_{context}"]
+= Verifying public network access
```

Comment:
Confirm why and ramifications here.

Reply:
@iamemilio @tomassedovic Say I don't do this. What happens, and would it be obvious that it's happening if I haven't completed this procedure?